docs/articles/faq/README.md

```diff
@@ -259,10 +259,22 @@ This error can occur in the following situations:
 
 Check if the Web and DNS servers are online and configured to accept requests to this URL.
 
+Before it throws an exception, TestCafe waits for *two minutes* for the webpage to be served. Use one of the following options to increase this timeout:
+
+* [`--page-request-timeout` CLI option](../documentation/reference/command-line-interface.md#--cache)
```

Comment:
```suggestion
* [`--page-request-timeout` (CLI)](../documentation/reference/command-line-interface.md#--cache)
```

Reply:
either add the definite article (*the* 'timeout' option) or get rid of the word 'option'. you're repeating it either way